vrs/vrs_schema_renaming_script_temp.py

```diff
@@ -0,0 +1,75 @@
+"""
+ Script to update column names from GA4GH VRS-Python Branch 0.8.1 to 0.8.2 convention
+ See: https://github.com/ga4gh/vrs-python/releases/tag/0.8.2
+ Note that this script will be run on 05-04-23 or 05-05-23 as a temporary fix
+ These changes will be permenantly added to the script which adds the VRS Annotations in a later update
+ Note that this code may not be fully productionized or up-to-date with the rest of the team's code standards 
+"""
+
+import argparse
+import hail as hl
+
+
+def main(args):
+
+    import_path = args.import_ht
+    ht = hl.read_table(import_path)
+    ht = ht.sample(0.01)
```

Comment:
am assuming this line is only here for testing?

Reply:
yep, I forgot to take that one out, my bad